### ISSUE TITLE
Quick Open screen reader should not only read file names but their relative paths as well

### ISSUE THREAD

**Issue by mltony** [microsoft/vscode#93657]
I am a screenreader user. In my company we have a pretty large mono repository, and when I try to open a file by name using Ctrl+P command (quick open)), most likely there are dozens of files by the same name in different directories displayed as suggestions. Visually VSCode displays the directory next to each suggestion, but when I tried it with NVDA, it only speaks the file name. Without directory this feature is virtually unusable, since I cannot figure out which suggestion is the right one. 
It would be great if VSCode could announce the relative directory of suggested files, or at least it would be great to have an option for this.
Thanks!
CC: @isidorn

**Comment by jvesouza** [comment]
Well noted. I tested using orca and the result was the same.

Another point is that I use an orca command called 'speaks the title bar. to find out which file is being edited. The command announces only the file name and not the full path.
I believe the problem will be the same, we will not be able to identify exactly which file is being edited.

**Comment by isidorn** [comment]
@mltony thanks for filling this. Currently we just read the filename since we got feedback from users that reading the relative path is too spammy.

Thus we can tackle this in one of the following ways:
1) We provide a setting for this which by default is false
2) We always read the relative file path at the end
3) We only read the relative path if there are multiple results with the same name
4) We try to use `aria-annotations` to annotate an element with a relative path. Though please note that aria annotations are work in progress.

I would only introduce the setting if none of the other options seem viable.

fyi @joanmarie for ideas

**Comment by bpasero** [comment]
@isidorn simply follow all users of [`IQuickPickItem.ariaLabel`](https://github.com/microsoft/vscode/blob/6f5e1edcd109df1184ce72bf90055f15223c406a/src/vs/base/parts/quickinput/common/quickInput.ts#L24) to find the places that need adjustment. The one for `Cmd+P` in particular is [here](https://github.com/microsoft/vscode/blob/6f5e1edcd109df1184ce72bf90055f15223c406a/src/vs/workbench/contrib/search/browser/anythingQuickAccess.ts#L837).

**Comment by bpasero** [comment]
Oh, there is also [`IQuickPick.ariaLabel`](https://github.com/microsoft/vscode/blob/6f5e1edcd109df1184ce72bf90055f15223c406a/src/vs/base/parts/quickinput/common/quickInput.ts#L201).

**Comment by jvesouza** [comment]
@isidorn  I like the option 2, We always read the relative file path at the end.

**Comment by isidorn** [comment]
@jvesouza also works for me and I think it makes sense since user can easily dismiss it. Let's start with that next week in Insiders so we gather feedback. Thanks
